test/app-for-e2e/tests/run-tests

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+AWS_DEFAULT_REGION="us-east-2"
+AWS_REGION=$AWS_DEFAULT_REGION
+DB_NAME="Pets"
+SERVER_URL="http://localhost:1738"
+NAME_REGEX='^[A-Z][a-z]*'
+EXIT_CODE_TO_RETURN=0
+
+export AWS_REGION
+export AWS_DEFAULT_REGION
+
+# Health checks
+function health_check() {
+  sleep 1
+  while true; do
+    echo "⏳ Waiting for server..."
+    health=$(curl -s "$1" || :)
+    # server is ready if ANY response is received
+    if [[ -n $health ]]; then
+      echo "======================================================================================================"
+      echo "👍 Server ready: $1 👍"
+      echo "======================================================================================================"
+      break
+    fi
+    sleep 1
+  done
+}
+
+function db_check() {
+  sleep 1
+  echo "Verifying database exists..."
+  aws dynamodb describe-table --table-name "$1" >/dev/null
+  status=$?
+  if [[ $status -ne 0 ]]; then
+    echo "❌ Database $DB_NAME needs to be created before running tests"
+    EXIT_CODE_TO_RETURN=1
+  fi
+  echo "✅ Verified Database $DB_NAME exists"
+}
+
+# Tests
+function create_pets_test() {
+  num_pups=1000
```

Comment:
Any suggestions for some starting values? I will consolidate this and the `delete_num` var in `delete_pets_test` as a const at the top of test file as the intent is to make sure table is in the same state it was in prior to kicking off the tests

Reply:
1,000 is probably too low, maybe starting at 10k. But I'm not really sure. You'll probably have to just see what the monitoring tells you